src/utils/index.ts

```diff
@@ -42,19 +42,27 @@ export function toStringNoMS(date: Date): string {
   return date.toISOString().replace(/\.[0-9]{3}Z/, 'Z')
 }
 
-export async function getFileInfo(url: string): Promise<FileMetadata> {
+export async function getFileInfo(
```

Comment:
perfect. Probably good time to move this out of this generic file. For the ocean.js-aquarius replacement methods we put them all into its own file so we could do that here too and put it into a new file `utils/provider.ts`. And then name the method exactly like ocean.js does to make the mimicking more obvious in code, so `fileinfo()`